packages/react-percy-storybook/src/index.js

```diff
@@ -38,38 +29,54 @@ function assertRtl(rtl) {
   }
 }
 
-export const percyAddon = {
-  addWithPercyOptions: function(storyName, options, storyFn) {
-    if (typeof options === 'function') {
-      storyFn = options;
-      options = undefined;
-    }
+export default function createPercyAddon() {
+  /**
+   * This map contains a mapping from stories for a kind to
+   * options of the respective story,
+   * looking like this:
+   * {
+   *   kindA: { story1: options, storyX: options },
+   *   kindB: { story...}
+   *   ...
+   * }
+   */
+  const optionsTuples = new Map();
 
-    if (options) {
-      assertWidths(options.widths);
-      assertRtl(options.rtl);
+  const percyAddon = {
+    addWithPercyOptions: function(storyName, options, storyFn) {
+      if (typeof options === 'function') {
+        storyFn = options;
+        options = undefined;
+      }
 
-      optionsTuples.set(this.kind, optionsTuples.get(this.kind) || new Map());
-      const tuplesForKind = optionsTuples.get(this.kind);
-      tuplesForKind.set(storyName, options);
-    }
-    this.add(storyName, context => storyFn(context));
-  },
-};
+      if (options) {
+        assertWidths(options.widths);
+        assertRtl(options.rtl);
 
-export const serializeStories = getStorybook => {
-  const storybook = getStorybook();
-  storybook.forEach(storyBucket => {
-    if (!optionsTuples.has(storyBucket.kind)) {
-      return;
-    }
-    const tuplesForKind = optionsTuples.get(storyBucket.kind);
-    storyBucket.stories.forEach(story => {
-      if (tuplesForKind.has(story.name)) {
-        story.options = tuplesForKind.get(story.name);
+        optionsTuples.set(this.kind, optionsTuples.get(this.kind) || new Map());
+        const tuplesForKind = optionsTuples.get(this.kind);
+        tuplesForKind.set(storyName, options);
       }
+      return this.add(storyName, storyFn);
+    },
+  };
+
+  const serializeStories = getStorybook => {
+    const storybook = getStorybook();
+    storybook.forEach(storyBucket => {
+      if (!optionsTuples.has(storyBucket.kind)) {
+        return;
+      }
+      const tuplesForKind = optionsTuples.get(storyBucket.kind);
+      storyBucket.stories.forEach(story => {
+        if (tuplesForKind.has(story.name)) {
+          story.options = tuplesForKind.get(story.name);
+        }
+      });
     });
-  });
-  if (typeof window === 'object') window[storiesKey] = storybook;
-  return storybook;
-};
+    if (typeof window === 'object') window[storiesKey] = storybook;
+    return storybook;
+  };
+
+  return { percyAddon, serializeStories };
```

Comment:
What node version are you running? It could be the shorthand that is the problem, I can refactor - I developed this in node 8, didn't think of older versions, which I should have, I guess.